content/setup.rst

```diff
@@ -81,16 +81,20 @@ The following commands should give version numbers::
    (mpi) $ mpicc --version
    (mpi) $ mpirun --version  
 
+With OpenMPI you can also try the ``-showme`` flag to see what the ``mpicc``
+compiler wrapper does under the hood::
+
+   (mpi) $ mpicc -showme
+
 To compile an MPI code `hello_mpi.c`, you should now be able to do::
 
   (mpi) $ mpicc -o hello_mpi.x hello_mpi.c
   (mpi) $ mpirun -n 2 hello_mpi.x
   
 To compile with OpenMP support for hybrid MPI+OpenMP codes, you need
-to add the `-fopenmp` flag and possibly explicitly link to the relevant library
-installed by `conda`::
+to add the ``-fopenmp`` flag and link to the `libomp` library::
 
-  (mpi) $ mpicc -o hello_omp_mpi.x hello_omp_mpi.c -L/path/to/miniconda3/envs/mpi/lib -lomp
+  (mpi) $ mpicc -fopenmp -o hello_omp_mpi.x hello_omp_mpi.c -lomp
```

Comment:
I'll remove the `-lomp` altogether, because it's `-lgomp` for GNU and `-liomp5` for Intel. And likely something different on macOS. We can help with linking against the OpenMP runtime during the workshop.

Reply:
ok that's probably best. clang (`-lomp`) might not always be what's installed by `conda install compilers`